plugin/lspconfig.lua

```diff
@@ -46,15 +48,23 @@ local get_clients_from_cmd_args = function(arg)
   return vim.tbl_values(result)
 end
 
+for group, hi in pairs {
+  LspInfoBorder = { link = 'Label', default = true },
+  LspInfoList = { link = 'Visual', default = true },
+  LspInfoTips = { link = 'Comment', default = true },
+} do
+  api.nvim_set_hl(0, group, hi)
+end
+
 -- Called from plugin/lspconfig.vim because it requires knowing that the last
 -- script in scriptnames to be executed is lspconfig.
-vim.api.nvim_create_user_command('LspInfo', function()
+api.nvim_create_user_command('LspInfo', function()
   require 'lspconfig.ui.lspinfo'()
 end, {
   desc = 'Displays attached, active, and configured language servers',
 })
 
-vim.api.nvim_create_user_command('LspStart', function(info)
+api.nvim_create_user_command('LspStart', function(info)
   local server_name = string.len(info.args) > 0 and info.args or nil
```

Comment:
For strings `#` and `string.len` are equivalent
```suggestion
  local server_name = #info.args > 0 and info.args or nil
```